### PR TITLE
Revert "add route"

### PR DESCRIPTION
**Pull request by TetyanaKarpikova**
Reverts TetyanaKarpikova/itrex-hw1#3

### CODE REVIEW

File: view/signup.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const SignUp = {
+    render: () => {
+        return `
+        <main class="main-wrapper">
+        <div class="image-background">
+            <img class="image-background__top" src="images/bg.jpg" alt="background images">
```
Comment (matthewlaz) [inline view/signup.js]: better practice would be if you use background property instead of div -> img

Comment (matthewlaz) [reply view/signup.js]: also routes for images is broken. I was have the same problem. My solution was :  set picture with backround property using css.

File: view/signup.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                    type="button">
+                </button>
+            </div>
+            <div class="wrapper-send-form">
+                <button class="wrapper-send-form__button" type="submit" data-name="/signin">Sign Up ></button>
+            </div>
+        </form>
+        <div class="wrapper-link">
+            <span> Already have an account? </span>
+            <a href="#" data-name="/signin">Sign in</a>
```
Comment (matthewlaz) [inline view/signup.js]: If I click this button i will redirect to sign up page, but I can't step back with help of my browser arrow navigation

**Review by matthewlaz — CHANGES_REQUESTED** [review]

File: css/styleSign.css [hunk 3]
```diff
@@ -144,8 +144,10 @@ body {
   background-image: url(../images/icon-eye-blocked.svg);
 }
 
-.wrapper-input__icon-close {
-  background-image: url(../images/icon-eye-blocked.svg);
+.wrapper-input__icon-eye {
+  opacity: 0.5;
```
Comment (Lemout17) [inline css/styleSign.css]: I don't understand the reason of use `opacity `property ?

Comment (TetyanaKarpikova) [reply css/styleSign.css]: downloaded the picture, it is too black and this parameters to make this picture look more gray on the page

**Pull request merged**